packages/xo-web/src/common/editable/index.js

```diff
@@ -359,6 +361,26 @@ export class Select extends Editable {
     }
   }
 
+  componentDidUpdate (prevProps, prevState) {
+    const { form } = this.state
+    if (prevState.form !== form) {
+      form !== undefined && form.addEventListener('focusout', this._closeForm)
+    }
+  }
+
+  componentWillUnmount () {
+    const { form } = this.state
+    form !== undefined && form.removeEventListener('focusout', this._closeForm)
+  }
+
+  _closeForm = event => {
+    setTimeout(() => {
+      if (document.body === document.activeElement) {
+        this._closeEdition()
```

Comment:
Please explain.

Reply:
when `Select`  loses focus I need to check if `Button` gains focus if not we close edition that's why I check if the active element is `body` to close edition.
an example why I don't use `onBlur`:
click on Select:  Select => focus
click on Button (select all):  Select => blur  Button => focus (thats why I don't use `onBlur` in `form`)

Reply:
[`editable#Size`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/editable/index.js#L484-L489) had the same issue. Please use the same workaround if needed and the same method names for homogeneity.